.gitignore

```diff
@@ -112,3 +113,6 @@ venv.bak/
 
 # Benchmarking results
 .asv/
+
+# VSCode
+.vscode/
```

Comment:
I can undo this addition, as well as the `pip-wheel-metadata` one before this PR is merged. It's easier for me to just `git add .` in the mean time.

Reply:
fwiw, you can also locally ignore files by editing your `.git/info/exclude` file
https://stackoverflow.com/questions/1753070/how-do-i-configure-git-to-ignore-some-files-locally

Reply:
TIL :)